CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contributing to Canopy
```

Comment:
can you add a link to this MD from the main MD?

Reply:
+1

There should be a "how to contribute" section in the main MD with a link to this doc